src/Runner.Worker/ActionManager.cs

```diff
@@ -683,7 +715,8 @@ private async Task BuildActionContainerAsync(IExecutionContext executionContext,
                         // * Policy validation failed
                         if (ex is WebApi.UnresolvableActionDownloadInfoException)
                         {
-                            throw;
+                            throw new WebApi.UnresolvableActionDownloadInfoException("Unable to resolve action download info", ex);
```

Comment:
What's the benefit of wrapping the exception with different messaging?  Does it make the problem easier to spot in telemetry?  or logs?

Reply:
+1, why do we need to wrap with the same exception type?

Reply:
My original intent was to make logs easier to read, but after reading up more on how c# handles exceptions, it looks like it's a waste of resources to wrap the exception. Removed!